Ticketing/Ticketing.DAL/Domains/Payment.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Ticketing.DAL.Domain
+{
+    public class Payment
+    {
+        public int Id { get; set; }
+        public decimal Amount { get; set; }
+        public int PaymentStatusId { get; set; }
+        public PaymentStatus? PaymentStatus { get; set; }
```

Comment:
can payment status be undifined? also looks like duplication between PaymentStatusId  and PaymentStatus

Reply:
done